packages/caip-stream/src/streams/SplitStream.ts

```diff
@@ -0,0 +1,23 @@
+import { Duplex } from 'readable-stream';
```

Comment:
Can you explain why this stream is necessary? Maybe it would be good to do a walk through of this PR because I haven't worked deeply with our streams plumbing!